### PR TITLE
Use rate instead of keep_prob calling dropout in keras

### PR DESCRIPTION
**Pull request by yongtang**
While running tf.keras I noticed the following warning:
```
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
```

This fix fixes the warning by replacing keep_prob with rate.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Review by fchollet — APPROVED** [review]
LGTM, thank you
